### ISSUE TITLE
Folded region audio cue sound too harsh

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#142016]
I've been using `audioCues.enabled` for the past few weeks, even though I may not be the primary audience for this, and I've really enjoyed the added sounds as it enhances my development workflow. 

However, I've found that the folded region audio cue sounds much too harsh compared to errors/breakpoints. To me, it sounds like someone fumbling through papers or writing on something and in the beginning this felt like I was still in a meeting 😄  see sample below (you'll need to turn on the 🔊 sound in the video clip):

https://user-images.githubusercontent.com/35271042/152199028-e22c15f1-b6ea-4417-a3c4-f87e892affea.mp4

Comparing the different sounds, the folded cue stands out by a lot and don't think it is warranted given the importance compared to errors/breakpoints. I think the sound should be more gentle and less harsh as it is less important. As I said, I may not be the primary user for this but just my feedback on the sound.

cc @gino-scarpino

**Comment by gino-scarpino** [comment]
Great feedback and ear @misolori. We were totally exploring a paper sound here, but we need to re-consider. I agree, especially on repetition, it's harsh. Align more with the other sounds. Added to my list.

Thank you!
________________________________

**Comment by boltex** [comment]
I cant seem to find the settings to choose a particular .wav file for which trigger, let alone the list of triggers...

**Comment by isidorn** [comment]
@boltex not supported yet.

**Comment by isidorn** [comment]
@gino-scarpino provided a new sound and @hediet added it to vscode, thus closing this as fixed

**Comment by isidorn** [comment]
@steverep thank you very much for your feedback, this is extremely valuable to us.

We agree that sound preference is something personal, so we might need to add more customisation for this in the future. You are suggesting customisation of each sound. While that might come in the future we will first introduce a control for the audio cue volume. I will let you know once we have that so you can try it out and let us know if this has helped in this particular case.

Looking forward to more feedback.

**Comment by cannona** [comment]
Ouch! @steverep is absolutely right here. I came here to file a bug that the folded code sound was not working at all. Turns out, I just had to turn off the speech of my screen reader to hear it.

I would argue that this sound should be more harsh, or at least more prominent than other sounds. This sound is the only reliable way for screen reader users to know that there is some folded code at the cursor. This might not sound like a big deal, until you realize what confusion might be caused by thinking that part of your code is missing, and not just hidden.

Needless to say, I'm pretty disappointed with this change. I realize that most of us are volunteers, but if at all possible, I would encourage the team to find ways to seek feedback from people who *are* members of the target audience for accessibility features before making changes like this in future. I would very much appreciate having the original behavior reinstated if there is no better solution for this in the short term.

Bug again, like @steverep, I am very grateful for this functionality, and for all the amazing efforts from the team relating to accessibility. In spite of the occasional complaint, such as the above, this application has truly changed my life as a developer. Thank you all!

**Comment by steverep** [comment]
> We agree that sound preference is something personal, so we might need to add more customisation for this in the future. You are suggesting customisation of each sound. While that might come in the future we will first introduce a control for the audio cue volume. I will let you know once we have that so you can try it out and let us know if this has helped in this particular case.

More customization is always welcome, but I think volume is unlikely to make much difference in this case.  As I and @cannona  said, if screen reader users are the target demographic for these cues, then the chosen sounds need to stand out relative to fast speech right before and right after the sound.  This new 1.65 sound would probably have to be near a dangerous volume to do that.

**Comment by isidorn** [comment]
@cannona @steverep great feedback! Thanks a lot.
Let me address your comment regarding encouraging team to find feedback. I think we are actually not so bad at this since a lot of our screen reader users are using VS Code insiders and we get feedback constantly before something gets pushed to stable. Having said that, this one somehow slipped and we plan to fine tune the sound to be something more like the previous one. I have created this issue for this https://github.com/microsoft/vscode/issues/144927
And I suggest that you switch to using VS Code insiders once we push the fix next week so you get unblocked here. Thanks

**Comment by cannona** [comment]
> I think we are actually not so bad at this since a lot of our screen reader users are using VS Code insiders and we get feedback constantly before something gets pushed to stable.

That's totally fair, and my words in this regard were, I think, poorly considered. You folks are far ahead of the vast majority of projects out there, and I should have taken greater care to not suggest otherwise.

Thank you again!

**Comment by isidorn** [comment]
@cannona thank you very much for the feedback and please keep it coming :)

**Comment by isidorn** [comment]
@steverep @cannona we are doing a survey to better understand our users and to design better default sounds. We would like to know if you are using headphones or speakers when working with VS Code?

Also I have opened this issue for discussion on sound customisations (nothing on the plan now, but we might tackle it in the future): https://github.com/microsoft/vscode/issues/145204

**Comment by cannona** [comment]
@isidorn I WFH, so mostly I'm using speakers. Also, audio folks have a way to measure the relative loudness of a sound. It's known as LUFS. You might look into it, as it may help you to balance things a bit better. As I understand it, it takes into account the average loudness of a sound, unlike most calculations, which look only at the peaks.

Thanks for the info.

**Comment by steverep** [comment]
@isidorn I use both - speakers in my home office and mostly headphones elsewhere for privacy.

Adding to @cannona's recommendation to look at LUFS,you can easily alter this using Audacity.  Here's the [manual page for loudness normalization](https://manual.audacityteam.org/man/loudness_normalization.html).

You could also go deeper and read up on psycho-acoustics 📚 🔉 🥼 .
